resources/views/fields/button.blade.php

```diff
@@ -3,6 +3,8 @@
         <button type="button"
                 class="{{ $class }}"
                 data-action="screen--base#targetModal"
+                data-async="{{ ($async ? 'true' : 'false') }}"
```

Comment:
Strict types allow you to make functions like this:

```
data-async="{{ var_export($async) }}"
```

Reply:
Fixed